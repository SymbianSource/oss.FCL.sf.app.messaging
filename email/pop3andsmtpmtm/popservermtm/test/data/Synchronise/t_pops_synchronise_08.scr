+OK demon POP3 server ready (demon.co.uk)
+OK username accepted
+OK password accepted 
+OK 5 6540
+OK 5 messages
1 1314
2 1400 
3 1275
4 1234
5 1317
.
+OK 5 messages
1 t_pops_synchronise:message002
2 t_pops_synchronise:message004
3 t_pops_synchronise:message005
4 t_pops_synchronise:message006
5 t_pops_synchronise:message007
.
-ERR Unknown Command "CAPA"
** Read cancelled
+OK delete 2 accepted
+OK disconnecting

