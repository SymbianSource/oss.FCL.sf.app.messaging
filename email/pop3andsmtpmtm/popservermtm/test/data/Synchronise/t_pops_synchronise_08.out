STAT
LIST
UIDL
CAPA
DELE 2
QUIT
